ExperimentManager1.py

```diff
@@ -1,43 +1,43 @@
-import time
-import time
-from Sound_manager import SoundManager
-from vimba import *
-from VideoAnalyser1 import Video_Analyzer
+# Micky: I suggest to return to the original file name. the '1' suffix is confusing
+
+from VideoAnalyzerSim import VideoAnalyzer
+#from VideoAnalyser1 import Video_Analyzer
 # from VideoAnalyzer import Video_Analyzer
+# from VideoAnalyzerStub import Video_Analyzer
+
+#from ArduinoDigital import ArduinoDigital
+from ArduinoDigitalSim import ArduinoDigital
 
 from MouseMonitor1 import MouseMonitor
-# from VideoAnalyzerStub import Video_Analyzer
 from MouseMonitor1 import Locations
-from logger import TrialLogger
+from SoundManager import *
 from simulated_mouse import Simulated_mouse
-from ArduinoDigital import ArduinoDigital
 from StateManager import StateManager
 from StateManager import States
 from StateManager import Events
-import experimentgui
 from RewardManager import RewardManager
-import tkinter as tk
 from logger import TrialLogger
 from OpponentType import OpponentType
 
 
-# from ArduinoDigitalSim import ArduinoDigital  ##Anushka-new class
 class ExperimentManager:
     def __init__(self):
         # Initialize other components here
-        self.root = tk.Tk()  # Create a new Tkinter root window if not provided
+        #self.root = tk.Tk()  # Create a new Tkinter root window if not provided   #Micky: The video Analyzer stub should create its own TK root
```

Comment:
made changes in latest version

Reply:
OK. Let's keep the comment until I see the changes.